src/Servers/Connections.Abstractions/ref/Microsoft.AspNetCore.Connections.Abstractions.netcoreapp3.0.cs

```diff
@@ -0,0 +1,202 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+namespace Microsoft.AspNetCore.Connections
+{
+    public partial class AddressInUseException : System.InvalidOperationException
```

Comment:
Does this mean the reference assemblies are generated with DotNetBuildFromSource enabled?